tests/puppeteer-utils.js

```diff
@@ -2,10 +2,19 @@ const puppeteer = require('puppeteer');
 require('dotenv').config();
 
 const BASE_URL = process.env.BASE_URL;
+const BASE_INTERVIEW_URL = `${BASE_URL}/interview?i=docassemble.playground${process.env.PLAYGROUND_ID}${process.env.PROJECT_NAME}`;
+const BRANCH = process.env.BRANCH
+    || (process.env.BRANCH_PATH && process.env.BRANCH_PATH.split('/')[2])
+    || 'master';
```

Comment:
This looks good to me, but causes a merge conflict with my code. I think the fix is just to add my const variable `PROJECT_NAME ` here.